### ISSUE TITLE
请教个问题！

### ISSUE THREAD

**Issue by wswdbl** [libjxl/libjxl#3488]
这个压缩有分压缩等级吗？需要怎么修改啊！

**Comment by jonsneyers** [comment]
In `cjxl`, you can adjust the quality/filesize trade-off using `--distance` (higher distance is more distortion) or `--quality` (higher quality is less distortion).

You can also adjust the computational effort (cpu time / speed) trade-off using `--effort`.

This information will be printed if you run `cjxl` without any arguments, or with `--help`.

**Comment by wswdbl** [comment]
那就是只能调整--distance这个参数的大小，决定图片的压缩质量吗？

**Comment by jonsneyers** [comment]
Yes, `--distance` is expressed in JND (just noticeable difference) units, so 0 is lossless, 1 is one JND, etc.

Alternatively you can use `--quality`, which internally gets mapped to a distance, e.g. quality 100 is lossless, quality 90 corresponds to distance 1.
